core/bin/zksync_tee_prover/src/tee_prover.rs

```diff
@@ -67,10 +67,24 @@ impl fmt::Debug for TeeProver {
 }
 
 impl TeeProver {
+    /// Signs the message in Ethereum-compatible format for on-chain verification.
+    pub fn sign_message(sec: &SecretKey, message: Message) -> Result<[u8; 65], TeeProverError> {
+        let s = SECP256K1.sign_ecdsa_recoverable(&message, sec);
```

Comment:
You can use [the `zksync_crypto_primitives` library](https://github.com/matter-labs/zksync-era/blob/main/core/lib/crypto_primitives/src/ecdsa_signature.rs); it has this functionality (and one used in the unit tests below) implemented.

Reply:
@slowli, kindly `ping`.

Reply:
The existing primitives are private because so far, there was no use case to make them public, and keeping stuff private by default is a good practice. Now that such a use case has appeared, I'd suggest to make the necessary primitives public and export them from `crypto_primitives`.